make/Images.gmk

```diff
@@ -108,6 +109,55 @@ $(eval $(call SetupExecute, jlink_jdk, \
 
 JLINK_JDK_TARGETS := $(jlink_jdk)
 
+ifeq ($(JLINK_PRODUCE_RUNTIME_LINK_JDK), true)
+  RL_BUILD_CLASSES := runtimeimagelinkdeltaproducer-classes
+  RL_DELTA_GEN_CLASSES := $(BUILDTOOLS_OUTPUTDIR)/$(RL_BUILD_CLASSES)
+
+  RL_JIMAGE_PATH_ARG := $(JDK_IMAGE_DIR)/lib/modules
+  RL_DIFFS_OUTPUT_FILE_ARG := $(JDK_IMAGE_DIR)/lib/runtime-image-link.delta
+  RL_MOD_PATH_ARG := $(IMAGES_OUTPUTDIR)/jmods
+  TOOL_RUNTIME_IMAGE_LINK_DELTA_PRODUCER := $(BUILD_JAVA) --add-modules jdk.jlink \
+      --add-exports=jdk.jlink/jdk.tools.jlink.internal.runtimelink=ALL-UNNAMED \
+      --add-exports=java.base/jdk.internal.module=ALL-UNNAMED \
+      --add-exports=java.base/jdk.internal.jimage=ALL-UNNAMED \
```

Comment:
These three are repeated in both compilation and runtime so could potentially be set in a variable to avoid the duplication.